client/device/duoadminpanel/create.go

```diff
@@ -0,0 +1,47 @@
+package duoadminpanel
+
+import (
+	"context"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/http"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/publicapi"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/retry"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/url"
+	"time"
+)
+
+type CreateInput struct {
+	Name           string   `json:"name"`
+	Host           string   `json:"host"`
+	IntegrationKey string   `json:"integrationKey"`
+	SecretKey      string   `json:"secretKey"`
+	Labels         []string `json:"labels"`
+}
+
+type CreateOutput = ReadOutput
+
+func Create(ctx context.Context, client http.Client, createInp CreateInput) (*CreateOutput, error) {
+
+	client.Logger.Println("creating duo admin panel")
+
+	createUrl := url.CreateDuoAdminPanel(client.BaseUrl())
+
+	transaction, err := publicapi.PostForTransaction(
```

Comment:
I would rename PostForTransaction to TriggerTransaction

Reply:
done 👍